### DISCUSSION TITLE
[Atlantis]Adding Multi-language support to routes

### DISCUSSION THREAD

**Discussion by kaanunver-zeneks** [primefaces/community#87] [PrimeReact Templates] [answered]
Usetranslation works fine with every other component but when I implement it in the labels in Routes.tsx and try to change the language it doesn't work properly. Items in the sidebar doesn't change with language change and when i refresh it said items appear as blanks, this persist through logging the user out and in. Oddly enough changing a line of code in routes and compiling it again fixes this problem until the language is changed again. Is there another, proper way of doing this?

**Comment by buraksaglam089** [comment, marked as answer]
Hello,

After the latest update of Atlantis React, it will be migrated to the next-gen using Next.js. We will revisit this issue when the new version is released.

Best regards.

**Comment by mertsincan** [comment]
Hi,

Please try the latest version. 

Best Regards,
